packages/ckeditor5-emoji/tests/emojipicker.js

```diff
@@ -150,6 +150,25 @@ describe( 'EmojiPicker', () => {
 		expect( originalFirstEmojiInGridTitle ).to.not.equal( newFirstEmojiInGridTitle );
 	} );
 
+	it( 'should store the category when the picker is closed', async () => {
```

Comment:
The category is not stored when the picker is closed, but when the category is changed.

```suggestion
	it( 'should load previous category after reopening the emoji picker', async () => {
```